src/components/Login.js

```diff
@@ -67,7 +70,7 @@ class Login extends Component {
             <label>
               Password:
               <input
-                type="password"
+                placeholder="Enter password"
```

Comment:
this type= password makes it so that the input is hidden when they type in the password (the dots) 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password
why remove it?